com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/HeliumEventProcessor.kt

```diff
@@ -0,0 +1,76 @@
+package com.chartboost.heliumsdk.unity
+
+import android.util.Log
+import com.chartboost.heliumsdk.ad.HeliumAdError
+import org.json.JSONException
+import org.json.JSONObject
+
+object HeliumEventProcessor {
+    private const val TAG = "HeliumEventProcessor"
+
+    @JvmStatic
+    fun serializeHeliumEvent(
+        placementName: String,
+        heliumAdError: HeliumAdError?,
+        eventConsumer: HeliumEventConsumer<String, Int, String>
+    ) {
+        eventConsumer.accept(placementName, heliumAdError?.getCode() ?: -1, heliumAdError?.getMessage() ?: "")
+    }
+
+    @JvmStatic
+    fun serializeHeliumBidEvent(
+        placementName: String,
+        dataMap: HashMap<String, String>,
```

Comment:
Would advise against including the data type in variable names. This could just be `data` and it'd still make sense, I think.

Reply:
true